Classes/S3Target.php

```diff
@@ -394,6 +394,9 @@ protected function getRelativePublicationPathAndFilename(ResourceMetaDataInterfa
     {
         if ($object->getRelativePublicationPath() !== '') {
             $pathAndFilename = $object->getRelativePublicationPath() . $object->getFilename();
+        } elseif (isset($this->s3DefaultProfile['minio']) && $this->s3DefaultProfile['minio']) {
+            // If MinIO is used we have to change to url style. Creating an object beneath an object is not supported
+            $pathAndFilename = $object->getSha1() . '_folder/' . $object->getFilename();
```

Comment:
@etlam Can you point to the Minio document discussing this limititation ? I use Minio for a few project with Flow ... don't know if i'm lucky but it work with just the path style configuration.

Reply:
https://github.com/minio/minio/issues/7335#issuecomment-470683398

Reply:
That is a valid discussion, but is it actually failing for you with?

Reply:
Yes, we use Sitegeist.Kaleidoscope in our project. I do not know if it would fail without Kaleidoscope.